packages/nx/src/tasks-runner/run-command.ts

```diff
@@ -319,25 +402,52 @@ async function ensureWorkspaceIsInSyncAndGetGraphs(
       extraOptions
     );
 
-    if (nxJson.sync?.applyChanges === true) {
-      spinner.succeed(`The workspace was synced successfully!
+    const successTitle = anySyncGeneratorsFailed
+      ? // the identified changes were synced successfully, but the workspace
+        // is still not up to date, which we'll mention next
+        'The identified changes were synced successfully!'
+      : // the workspace is fully up to date
+        'The workspace was synced successfully!';
+    const successSubtitle =
+      nxJson.sync?.applyChanges === true
+        ? 'Please make sure to commit the changes to your repository or this will error on CI.'
```

Comment:
```suggestion
        ? 'Please make sure to commit the changes to your repository or this will error in CI.'
```

Reply:
I just feel like "in" is the more common preposition here, e.g. https://www.npmjs.com/package/is-in-ci